src/jsonpath/parser.c

```diff
@@ -64,15 +58,15 @@ const char* AST_STR[] = {
     "AST_NODE_LIST",   /**/
     "AST_NULL",        /**/
     "AST_OR",          /**/
-    "AST_PAREN_LEFT",  /**/
-    "AST_PAREN_RIGHT", /**/
     "AST_RECURSE",     /**/
     "AST_RGXP",        /**/
     "AST_ROOT",        /**/
     "AST_SELECTOR",    /**/
     "AST_WILD_CARD",   /**/
 };
 
+bool is_unary(enum ast_type type) { return type == AST_NEGATION; }
```

Comment:
Formatting: function body on its own line would be more readable.

Reply:
Same as above, this is automatically formatted by clang-format. Will look into updating the defaults.